cartographer/mapping/pose_graph.h

```diff
@@ -133,6 +133,10 @@ class PoseGraph : public PoseGraphInterface {
   virtual std::map<std::string /* landmark ID */, PoseGraph::LandmarkNode>
   GetLandmarkNodes() = 0;
 
+  // Sets landmark pose.
```

Comment:
Maybe be a bit more verbose: 

`Sets global pose of landmark 'landmark_id' to given 'global_pose'.`

Reply:
ok, will do